src/Annotator.ts

```diff
@@ -1,5 +1,5 @@
 interface Annotator {
-    start(manifestUrl: string): Promise<void>;
+    start(): Promise<void>;
```

Comment:
I created an issue for doing this for all the interfaces https://github.com/NYPL-Simplified/webpub-viewer/issues/5